libfuse/fs.go

```diff
@@ -199,18 +199,22 @@ func (f *FS) Root() (fs.Node, error) {
 
 // Statfs implements the fs.FSStatfser interface for FS.
 func (f *FS) Statfs(ctx context.Context, req *fuse.StatfsRequest, resp *fuse.StatfsResponse) error {
-	// TODO: Fill in real values for these.
-	var bsize uint32 = 32 * 1024
 	*resp = fuse.StatfsResponse{
-		Blocks:  ^uint64(0) / uint64(bsize),
-		Bfree:   ^uint64(0) / uint64(bsize),
-		Bavail:  ^uint64(0) / uint64(bsize),
-		Files:   0,
-		Ffree:   0,
-		Bsize:   bsize,
+		Bsize:   fuseBlockSize,
 		Namelen: ^uint32(0),
-		Frsize:  0,
+		Frsize:  fuseBlockSize,
 	}
+	status, _, err := f.config.KBFSOps().Status(ctx)
```

Comment:
Ok, good that there are none in an idle state.  Though now I'm worried that we'll be blocking foreground operations on an unnecessary server call once every 5 seconds.  Once journaling is on, this will be pretty noticeable.  (Not sure if those `statfs` calls actually block the other Finder operation or if they're off the critical path.)

Maybe we can do something where we trigger a background RPC to update the cache, but still serve it from the cache if it's less than 2*timeout seconds old?